### PR TITLE
stiffness function for an orthotropic material

### PR DESCRIPTION
**Pull request by vlukes**

### CODE REVIEW

File: sfepy/mechanics/matcoefs.py [hunk 1]
```diff
@@ -146,6 +146,51 @@ def youngpoisson_from_stiffness(stiffness, plane='strain'):
 
     return young, poisson
 
+
+def stiffness_from_youngpoissonshear_ortho3(young, poisson, shear):
+    r"""
+    Compute 3D stiffness tensor :math:`{\bm D}` of a orthotropic linear
```
Comment (rc) [inline sfepy/mechanics/matcoefs.py:152]: an orthotropic

File: sfepy/mechanics/matcoefs.py [hunk 2]
```diff
@@ -146,6 +146,51 @@ def youngpoisson_from_stiffness(stiffness, plane='strain'):
 
     return young, poisson
 
+
+def stiffness_from_youngpoissonshear_ortho3(young, poisson, shear):
```
Comment (rc) [inline sfepy/mechanics/matcoefs.py:150]: The name follows the system, but seems too long. What about `stiffness_from_yps_ortho3` or something like that? BTW. What does the "3" mean?

Comment (vlukes) [reply sfepy/mechanics/matcoefs.py]: "3" indicates that the function is only for 3D problems.

**Comment by rc** [comment]
OK, thanks!

**Pull request merged**